--- sass_bundle.js	2024-05-31 18:56:50
+++ sass_bundle.js	2024-05-31 19:01:51
@@ -65411,7 +65411,7 @@
         t2 = t1._string_scanner$_position,
         t3 = new A.StringBuffer(""),
         buffer = new A.InterpolationBuffer(t3, A._setArrayType([], type$.JSArray_Object));
-      for (t4 = t1.string, t5 = t4.length, t6 = !omitComments; true;)
+      breakout: for (t4 = t1.string, t5 = t4.length, t6 = !omitComments; true;)
         $label0$0: {
           _1_0 = t1.peekChar$0();
           if (92 === _1_0) {
@@ -65444,13 +65444,13 @@
           }
           if (13 === _1_0 || 10 === _1_0 || 12 === _1_0) {
             if (_this.get$indented())
-              break;
+              break breakout;
             t7 = A.Primitives_stringFromCharCode(t1.readChar$0());
             t3._contents += t7;
             break $label0$0;
           }
           if (33 === _1_0 || 59 === _1_0 || 123 === _1_0 || 125 === _1_0)
-            break;
+            break breakout;
           if (117 === _1_0 || 85 === _1_0) {
             t7 = t1._string_scanner$_position;
             if (!_this.scanIdentifier$1("url")) {
@@ -65472,7 +65472,7 @@
             break $label0$0;
           }
           if (_1_0 == null)
-            break;
+            break breakout;
           t7 = _this.lookingAtIdentifier$0();
           if (t7) {
             t7 = _this.identifier$0();
@@ -119279,7 +119279,7 @@
         t2 = t1._string_scanner$_position,
         t3 = new A.StringBuffer(""),
         buffer = new A.InterpolationBuffer0(t3, A._setArrayType([], type$.JSArray_Object));
-      for (t4 = t1.string, t5 = t4.length, t6 = !omitComments; true;)
+      breakout: for (t4 = t1.string, t5 = t4.length, t6 = !omitComments; true;)
         $label0$0: {
           _1_0 = t1.peekChar$0();
           if (92 === _1_0) {
@@ -119312,13 +119312,13 @@
           }
           if (13 === _1_0 || 10 === _1_0 || 12 === _1_0) {
             if (_this.get$indented())
-              break;
+              break breakout;
             t7 = A.Primitives_stringFromCharCode(t1.readChar$0());
             t3._contents += t7;
             break $label0$0;
           }
           if (33 === _1_0 || 59 === _1_0 || 123 === _1_0 || 125 === _1_0)
-            break;
+            break breakout;
           if (117 === _1_0 || 85 === _1_0) {
             t7 = t1._string_scanner$_position;
             if (!_this.scanIdentifier$1("url")) {
@@ -119340,7 +119340,7 @@
             break $label0$0;
           }
           if (_1_0 == null)
-            break;
+            break breakout;
           t7 = _this.lookingAtIdentifier$0();
           if (t7) {
             t7 = _this.identifier$0();
